core/components/Inputs/Select/Select.vue

```diff
@@ -308,6 +308,7 @@ export default {
         parsedInputValue() {
             let parsedInput = this.selectedOptions;
 
+            if (!process.client) return '';
```

Comment:
Why Select has context of business? And why it runs only on client side? isGrid? It cannot have information about any component from the app.